lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -243,8 +245,12 @@ private function getLastModification(ReflectionClass $class)
 
         $lastModification =  max(array_merge(
             [$filename ? filemtime($filename) : 0],
-            array_map([$this, 'getTraitLastModificationTime'], $class->getTraits()),
-            array_map([$this, 'getLastModification'], $class->getInterfaces()),
```

Comment:
What made us do this kind of change? It's missing from list provided at https://github.com/doctrine/annotations/pull/350#issuecomment-672222095. I don't like this unnecessary verbose syntax, what was wrong about previous one?

Reply:
It is in the list:

>  259 | ERROR | Class CachedReader contains unused private method getTraitLastModificationTime(). (SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod)

It's a bit verbose indeed, but [this thread](https://github.com/slevomat/coding-standard/issues/159) convinced me to go ahead.